docs/notebooks/bsync_examples/Primary-School-Levels1-2.md

```diff
@@ -0,0 +1,8205 @@
+            </Space>
+          </Spaces>
+        </ThermalZone>
+      </ThermalZones>
+    </Section>
+    
+
+
+#### 6.2.5 Other
+
+We add final elements that were missing from the L100 audit.
+
+
+
+```python
+b1 += bsync.TotalExteriorAboveGradeWallArea(18538.86)
+b1 += bsync.TotalExteriorBelowGradeWallArea(0.)
+b1 += bsync.OverallWindowToWallRatio(0.35)
+b1 += bsync.OverallDoorToWallRatio(0.1)
+b1 += bsync.BuildingAutomationSystem(True)
+
+b1_gfa += bsync.ExcludedSectionIDs(bsync.ExcludedSectionID(IDref=wb_sec["ID"]))
+b1_cfa += bsync.ExcludedSectionIDs(bsync.ExcludedSectionID(IDref=wb_sec["ID"]))
+```
+
+### 6.3 HVAC Systems, Central Plants, Air and Water Distribution Systems and Outside Air Control
+
+This section follows the requirements of Standard 211, section 6.2.1.3.
+
+The primary school has 4 VAV systems and 3 CAV systems.
+
+#### 6.3.1 Central Plant and VAV systems
+
+We will go step by step for the first VAV. We pick up where we left it for the level 1 audit, where we already defined:
+- the system type
+- the linked premises
+
+We add the missing elements. The VAV system comprises of:
+- A mixing box for return and outside air
+- a two-speed DX cooling coil
+- a water heating coil
+- a splitter, and a duct leading to each zone
+- a hot water reheat coil for each zone
+
+With BSyncPy, you can simply add information to existing elements. Hence, we can use the previously-defined vav_pod1, vav_pod2, vav_pod3 and vav_other variables and enrich them with additional children elements.
+
+Since the heating coils are supplied with hot water coming from a central plant, we define a central heating plant and its pump. We place it under one of the VAV systems, here we use the Pod 1 VAV.
+
+
+```python
+vav_pod1_plants = bsync.Plants()
+vav_pod1 += vav_pod1_plants
+
+vav_pod1_plants += bsync.HeatingPlants(
+    bsync.HeatingPlant(
+        bsync.HeatingPlantCondition("Good"),
+        bsync.Boiler(
+            bsync.BoilerType('Hot water'),
+            bsync.BurnerControlType("On Off"),
+            bsync.BurnerQuantity(1),
+            bsync.InputCapacity(431299.),
+            bsync.Boiler.Capacity(323474.),
+            bsync.CapacityUnits('W'),
+            bsync.AnnualHeatingEfficiencyValue(0.75),
+            bsync.AnnualHeatingEfficiencyUnits('COP'),
+            bsync.ThermalEfficiency(0.8),
+            bsync.BoilerLWT(155.),
+            bsync.CondensingOperation("Unknown"),
+            bsync.HeatingStaging("Variable"),
+            bsync.CombustionEfficiency(1.),
+            bsync.Quantity(1)
+        ),
+        bsync.HeatingPlant.ControlSystemTypes(
+            bsync.ControlSystemType(
+                bsync.ControlSystemType.Other(
+                    bsync.OtherCommunicationProtocolName("Unknown")
+                )
+            )
+        ),
+        bsync.YearInstalled(2018),
+        bsync.PrimaryFuel('Natural gas'),
+        bsync.BuildingAutomationSystem(Tr
```

Comment:
Getting error message: "TypeError: integer expected"

Reply:
On my end everything works fine, can you tell me which particular line is expecting an integer please?

![Screen Shot 2022-06-29 at 2 20 23 PM](https://user-images.githubusercontent.com/56871007/176537388-25452733-7e3d-454e-a968-a6bb425387d8.png)

Reply:
Line 8067: "pom_led += bsync.AnnualDemandSavingsCost(872.57)" is where I got the error.  If i change it to 872 it worked.

Reply:
I see it now! Sorry, somehow it was showing this comment for lines 6666-6669.